kombu/transport/redis_cluster.py

```diff
@@ -511,6 +522,7 @@ class Transport(RedisTransport):
 
     driver_type = 'redis-cluster'
     driver_name = driver_type
+    can_parse_url = True
```

Comment:
Is this property being used?

Reply:
Yes, it's used at https://github.com/sendbird/kombu/blob/75027490c71b83342f92b5293461c679233dc25e/kombu/connection.py#L675

Reply:
Ah I see. Thanks